ci-operator/step-registry/gather/aws-console/gather-aws-console-commands.sh

```diff
@@ -35,6 +35,8 @@ REGION="$(jq -r .aws.region "${SHARED_DIR}/metadata.json")"
 cat "${TMPDIR}/node-provider-IDs.txt" | sort | uniq | while read -r INSTANCE_ID
 do
 	echo "Gathering console logs for ${INSTANCE_ID}"
-	aws --region "${REGION}" ec2 get-console-output --instance-id "${INSTANCE_ID}" --output text > "${ARTIFACT_DIR}/${INSTANCE_ID}" &
+	aws --region "${REGION}" ec2 get-console-output --instance-id "${INSTANCE_ID}" --output text > "${ARTIFACT_DIR}/${INSTANCE_ID}-console" &
+	wait "$!"
+	aws --region "${REGION}" ec2 describe-instances --instance-id "${INSTANCE_ID}" --output json > "${ARTIFACT_DIR}/${INSTANCE_ID}-status.json" &
```

Comment:
Why add the JSON dump?  If there is some information it adds, can we drop the text dump?  Or do we need both?  And can the motivation for any changes here be explained in the commit message, where future devs can `blame` them?

Reply:
Split this to  https://github.com/openshift/release/pull/11814